x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/conditional/Case.java

```diff
@@ -66,7 +66,7 @@ public Case(
         Source source,
         @Param(name = "condition", type = { "boolean" }) Expression first,
         @Param(
-            name = "rest",
+            name = "value",
```

Comment:
Can we indicate in the variable name its "positive" scenario?

```suggestion
            name = "trueValue",
```